src/main/java/com/scribassu/tracto/dto/web/EducationFormListDto.java

```diff
@@ -0,0 +1,21 @@
+package com.scribassu.tracto.dto.web;
+
+import com.scribassu.tracto.domain.EducationForm;
+import lombok.Data;
+
+@Data
+public class EducationFormListDto {
+    String[] shortForms = new String[3];
```

Comment:
Dto should contain list of inner dtos with only one shortForm and fullForm, not arrays. EducationFormListDto should contain list of EducationFormDto. EducationFormDto should contain shortForm and fullForm.